App.js

```diff
@@ -1,18 +1,150 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { Alert, AppRegistry, Button, StyleSheet, Text, TextInput, View } from 'react-native';
+import { StackNavigator } from 'react-navigation';
+import Picker from 'react-native-picker';
+
+// export default class LogIn extends React.Component {
+//   // static navigationOptions = {
+//   //   title: 'Sign Up',
+//   // };
+//   constructor(props) {
+//     super(props);
+//     this.state = {text: ''};
+//   }
+//
+//   render() {
+//     return (
+//       <View>
+//         <View style={{padding: 10}}>
+//           <TextInput
+//             style={{height: 40}}
+//             placeholder="Username"
+//           />
+//           <Text style={{padding: 10, fontSize: 42}}>
+//           </Text>
+//         </View>
+//         <View style={{padding: 10}}>
+//           <TextInput
+//             style={{height: 40}}
+//             placeholder="Password"
+//           />
+//           <Text style={{padding: 10, fontSize: 42}}>
+//           </Text>
+//         </View>
+//         <Button
+//           onPress={() => { Alert.alert('Account Created!')}}
```

Comment:
We should not be using alerts (generally quite disruptive to the user experience. onPress should call a function called "onLoginPress", which can be empty for now. On successful login, we can just redirect to the sign up to debate page